docs/develop/03_workshop/04_attester/03_ctype.md

```diff
@@ -14,75 +14,86 @@ import GenerateCtype from '!!raw-loader!@site/code_examples/sdk_examples/src/wor
 | -------------| ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `$id`        | The KILT id of this CType. It is the most important property as it represents the **digital footprint** of the CType.                                               |
+| `$id`        | The KILT id of this CType. It's the most important property as it represents the **digital footprint** of the CType.                                               |
 | `$schema`    | A reference to the meta-schema describing what a CType may look like. There are two versions.                                                              |
 | `title`      | The title of the CType.                                                                                                                                             |
 | `properties` | The properties that a claim conforming to this CType may have.                                                                                                      |
 | `type` | Type is an object for all CTypes.                                                                                                  |
-| `additionalProperties` | The default is set to false. This restricts unwanted properties in a claim.                                                                                                      |
+| `additionalProperties` | The default is false. This restricts unwanted properties in a claim.                                                                                                      |
 
 A CType is stored on the KILT blockchain.
 
-In a real-life setup, a user would simply retrieve an existing CType from the chain or a repository of CTypes for example via a Credential Registry's REST API.
+<!-- TODO: What is a Credential Registry's REST API? -->
+In a real-world situation, a user would retrieve an existing CType from the chain or a repository of CTypes.
+For example, via a Credential Registry's REST API.
 
-In this tutorial, we'll have the <span className="label-role attester">Attester</span> create and attempt to store a CType on the KILT test blockchain.
+In this tutorial, the <span className="label-role attester">Attester</span> creates and attempts to store a CType on the KILT test blockchain.
 
 ## Create CType
 
-Copy the following to create a `CType` with a given schema:
+Copy the following that defines a `CType` with a given schema:
 
 <TsJsBlock fileName="attester/ctypeSchema">
   {CtypeSchema}
 </TsJsBlock>
 
 ## Get CType
 
+Copy the following to create a `CType` on the chain:
+
 <TsJsBlock fileName="attester/generateCtype">
   {GenerateCtype}
 </TsJsBlock>
 
-We'll use this to check if the `CType` is on-chain already.
-If yes we'll return it, otherwise we'll store it on-chain.
+<!-- TODO: Already exists and code hangs -->
```

Comment:
The CType already exists and the code hangs, which seems reasonable, but what can we do about it?